apps/java-api/src/main/java/org/scoalaonline/api/model/Song.java

```diff
@@ -0,0 +1,106 @@
+package org.scoalaonline.api.model;
+
+import javax.persistence.*;
+import java.util.Date;
```

Comment:
Date is deprecated, try using LocalDate
```suggestion
import java.time.LocalDate;
```